pocs/utils/images/__init__.py

```diff
@@ -87,12 +90,34 @@ def make_pretty_image(fname, timeout=15, **kwargs):  # pragma: no cover
 def _make_pretty_from_fits(fname, **kwargs):
     config = load_config()
 
-    title = '{} {}'.format(kwargs.get('title', ''), current_time().isot)
+    title = '{} {}'.format(kwargs.get('title', ''), current_time(pretty=True).isot)
```

Comment:
Tests are failing on this. Doing the `pretty=True` turns it into a string, so then you can't call `.isot` on it. You want one or the other. They are basically the same except `pretty` has spaces for easier reading.